### ISSUE TITLE
Command to encrypt strings

### ISSUE THREAD

**Issue by 3rj** [StringCare/stringcare#6]
Hi

Is there any command to auto encrypt all strings in dart files?

for example:

`flutter pub run stringcare:obfuscate`

search and encrypt all strings in project folder

**Comment by efraespada** [comment]
No, `stringcare:obfuscate` only works over the `assets` and `lang` (base) folders.

But it could be interesting to detect and extract all strings of a project in a lang file.
